app/src/app/utils/api/apiHandlers.ts

```diff
@@ -132,6 +133,11 @@ export interface ZonePopulation {
   total_pop: number;
 }
 
+
+export let populationAbortController: AbortController | null = null;
+export let updateAbortController: AbortController | null = null;
+export let currentHash: string = '';
```

Comment:
I: Approving to get this bug fix in but wanted to flag that tanstack query automatically can [provide a signal to each query function](https://tanstack.com/query/v4/docs/framework/react/guides/query-cancellation)! This might be a nicer pattern than maintaining a growing list of globals in `apiHandlers`

If using the tanstack built-in functionality seems like a good pattern, I'd recommend leaving a `// TODO: ...` for the next person here!

Reply:
Added!